test/integration/tests/changeauth.sh

```diff
@@ -48,6 +53,7 @@ tpm2_changeauth -o $ownerPasswd -e $endorsePasswd -l $lockPasswd
 
 tpm2_changeauth -O $ownerPasswd -E $endorsePasswd -L $lockPasswd -o $new_ownerPasswd -e $new_endorsePasswd -l $new_lockPasswd
 
-tpm2_clear -L $new_lockPasswd
```

Comment:
I wonder if we should leave this too to make sure that there's no regression when not providing the password with a file. I think that calling clear twice won't be an issue.

Reply:
That seems reasonable, I'll add this back in.

Reply:
I had to add in an extra *tpm2_changeauth* before making an extra *tpm2_clear*

Reply:
> I had to add in an extra _tpm2_changeauth_ before making an extra _tpm2_clear_

Sounds good to me, I think the more coverage the better.